provisioning/roles/php-xdebug/tasks/not-debian-7.0.yml

```diff
@@ -5,7 +5,7 @@
     - { src: xdebug.ini.j2, dest: "{{ etc_php_path }}/mods-available/xdebug.ini" }
 
 - name: Activate PHP configuration files
-  command: "{{phpenmod}} {{ item }}"
+  command: "{{php_phpenmod}} {{ item }}"
```

Comment:
Since you changed this line, please use the opportunity to add whitespaces around the braces. ;) `{{ php_phpenmod }}`